pyteal/ast/router.py

```diff
@@ -302,11 +287,13 @@ def wrap_handler(
                   passed in ABIReturnSubroutine and logged, then approve.
         """
         if not is_method_call:
+            wrap_to_name = "bare appcall" if wrap_to_name is None else wrap_to_name
+
             match handler:
                 case Expr():
                     if handler.type_of() != TealType.none:
                         raise TealInputError(
-                            f"bare appcall handler should be TealType.none not {handler.type_of()}."
+                            f"{wrap_to_name} handler should be TealType.none not {handler.type_of()}."
```

Comment:
`wrap_to_name` is optional but you use it in the error string. Does that print "None"?

Reply:
no, `wrap_to_name` is `None`, then converted to a default `"bare appcall"` on line https://github.com/algorand/pyteal/pull/636/files#diff-3844cf2bef1f9fc5ce0b421c65141cf76d172d885b5792ac0cd79dc360618769R290